src/helpers/releasesInfo.js

```diff
@@ -9,17 +9,29 @@ export const retrieveReleases = gitHubURL =>
                 constants.GITHUB_API_RELEASE_URL.replace(
                     "{repo_name}",
                     repositoryName
-                )
+                ),
+                {
+                    headers: {
+                        "If-None-Match": localStorage.getItem(constants.LOCAL_STORAGE.GET_RELEASES_ETAG) || ""
+                    }
+                }
             )
             .then(response => {
                 if (response.status === 200) {
+                    localStorage.setItem(constants.LOCAL_STORAGE.GET_RELEASES_ETAG, response.headers.etag.toString());
```

Comment:
I am wondering if we should clear the localStorage on unmount of the app? Not sure.